### PR TITLE
Give Captains CQC inside of their office

### PR DESCRIPTION
**Pull request by chefferz**
Does as it says on the tin.

Gives Captains CQC within their office. Pretty cut and dry.

### CODE REVIEW

**Comment by CameronWoof** [comment]
**BLOCKER - VERY IMPORTANT - DO NOT MERGE UNTIL RESOLVED**

Is this "Captain's Quarters Combat" or "Close Quarters Captaincy"?

**Comment by chefferz** [comment]
> 
> 
> **BLOCKER - VERY IMPORTANT - DO NOT MERGE UNTIL RESOLVED**
> 
> Is this "Captain's Quarters Combat" or "Close Quarters Captaincy"?

Renamed  Close Quarters Captaincy

**Comment by CameronWoof** [comment]
ok this is merge-ready

**Comment by GrayRachnid** [comment]
:joy:

File: code/modules/jobs/job_types/captain.dm [hunk 1]
```diff
@@ -62,4 +62,11 @@
...
-	suit_store = /obj/item/tank/internals/oxygen
\ No newline at end of file
+	suit_store = /obj/item/tank/internals/oxygen
+	
+/datum/outfit/job/captain/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE, client/preference_source)
+    ..()
+    if(visualsOnly)
+        return
+	var/datum/martial_art/cqc/captainship/bossman = new
+    bossman.teach(H)
```
Comment (Linzolle) [inline code/modules/jobs/job_types/captain.dm]: missing indent

File: code/modules/jobs/job_types/captain.dm [hunk 2]
```diff
@@ -62,4 +62,11 @@
...
 	suit = /obj/item/clothing/suit/space/hardsuit/captain
-	suit_store = /obj/item/tank/internals/oxygen
\ No newline at end of file
+	suit_store = /obj/item/tank/internals/oxygen
+	
+/datum/outfit/job/captain/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE, client/preference_source)
+    ..()
+    if(visualsOnly)
+        return
+	var/datum/martial_art/cqc/captainship/bossman = new
```
Comment (Linzolle) [inline code/modules/jobs/job_types/captain.dm:71]:
```suggestion
		var/datum/martial_art/cqc/captainship/bossman = new
```

File: code/modules/jobs/job_types/captain.dm [hunk 3]
```diff
@@ -62,4 +62,11 @@
...
+    	bossman.teach(H)
```
Comment (Linzolle) [inline code/modules/jobs/job_types/captain.dm]:
```suggestion
		bossman.teach(H)
```

File: code/modules/jobs/job_types/captain.dm [hunk 4]
```diff
@@ -62,4 +62,11 @@
...
 	suit = /obj/item/clothing/suit/space/hardsuit/captain
-	suit_store = /obj/item/tank/internals/oxygen
\ No newline at end of file
+	suit_store = /obj/item/tank/internals/oxygen
+	
+/datum/outfit/job/captain/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE, client/preference_source)
+    ..()
+    if(visualsOnly)
+        return
+		var/datum/martial_art/cqc/captainship/bossman = new
```
Comment (Linzolle) [inline code/modules/jobs/job_types/captain.dm]:
```suggestion
	var/datum/martial_art/cqc/captainship/bossman = new
```
what the fuck

**Comment by ghost** [comment]
is there a reason for this? chef got it because refs to Under Siege

**Comment by chefferz** [comment]
this might be a preview to the great pokemon gym battle update where all the heads get CQC in their office. 

as of now it was just a floated idea to test. likely going to be barely used but still something that would add an amusing dynamic.

**Comment by davidchan13** [comment]
Next we should give it to the hop when they are holding Ian.

Also not calling it Close Quarters Commands

**Comment by Ghommie** [comment]
"_Under Siege 2_ represents a steep comedown from its predecessor" - Rotten Tomatoes

**Pull request closed without merging**